src/main/java/nl/nn/testtool/web/api/MetadataApi.java

```diff
@@ -97,7 +101,7 @@ public Response getMetadataList(@PathParam("storage") String storageName,
 
 			return Response.ok().entity(metadata).build();
 		} catch (Exception e) {
-			return Response.status(Response.Status.INTERNAL_SERVER_ERROR).entity("Could not find metadata with limit [" + limit + "] and filter [" + filterParam + "] - detailed error message - " + e + Arrays.toString(e.getStackTrace())).build();
+			return Response.status(Response.Status.INTERNAL_SERVER_ERROR).entity("Could not find metadata with limit [" + limit + "] and filter [" + filterParams + "] - detailed error message - " + e + Arrays.toString(e.getStackTrace())).build();
```

Comment:
What does the `filterParams` return when it gets changed to a string? Is it something like `[test1, test2]` or `Object@3234` or something? If the latter is the case, it should first be converted.

Reply:
it would print it out as [test1, test2], but I did notice that it was still adding [] in front and behind it so I removed it.

Reply:
Perfect!